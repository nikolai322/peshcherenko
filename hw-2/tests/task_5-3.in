4
10 100 1000 10000
2
0.0001 -0.0002
