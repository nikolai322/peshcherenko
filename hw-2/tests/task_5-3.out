11110
-0.000100
